### ISSUE TITLE
duplicate folders

### ISSUE THREAD

**Issue by huntj88** [esafirm/android-image-picker#65]
with this configuration

```
ImagePicker.create(this)
                    .single()
                    .showCamera(true)
                    .returnAfterFirst(true)
                    .enableLog(true)
                    .folderMode(true)
                    .start(CAMERA_RQ);
```

clicking the camera and then backing out of it will result in duplicate folders.

The picture below is after starting the camera and backing out multiple times

![image](https://user-images.githubusercontent.com/10103298/28535958-700a127a-7063-11e7-99fa-5067539c073e.png)

**Comment by esafirm** [comment]
Will check on this

**Comment by lyaotian** [comment]
It cause by the code here: https://github.com/esafirm/android-image-picker/blob/master/imagepicker/src/main/java/com/esafirm/imagepicker/adapter/FolderPickerAdapter.java#L57 

After capture images the ContentObserver(created in ImagePickerActivity) onChange()  called, and ImageLoadRunnable would run, when ImageLoadRunnable finish running, it would add all the folders to FolderPickerAdapter List<Folder> folders  by setData() method

**Comment by esafirm** [comment]
Fixed in `1.7.5` please kindly check 🙏
